code/game/area/almayer.dm

```diff
@@ -0,0 +1,383 @@
+//ALMAYER AREAS--------------------------------------//
+// Fore = West  | Aft = East //
+// Port = South | Starboard = North //
+/area/almayer
```

Comment:
These no longer exist and shouldn't be used, it's `mainship` now I belive in another file